net/pfSense-pkg-frr/files/usr/local/pkg/frr/frr_ospf6_interfaces.xml

```diff
@@ -184,6 +188,22 @@
 			<description>Dead Timer for this OSPF6 interface in seconds (Default 40).</description>
 			<type>input</type>
 		</field>
+
+		<field>
+			<name>BFD</name>
+			<type>listtopic</type>
+		</field>
+		<field>
+			<fielddescr>BFD Enable</fielddescr>
+			<fieldname>bfdenable</fieldname>
+			<description>
+				<![CDATA[
+				Listen for BFD events on peers created on the interface.
+				Every time a new neighbor is found a BFD peer is created to monitor the link status for fast convergence.
```

Comment:
Reword this a bit. Perhaps "When OSPF6 finds a new neighbor, it creates a BFD peer to monitor..."